src/Arcus.Scripting.KeyVault/Arcus.Scripting.KeyVault.psm1

```diff
@@ -76,10 +76,10 @@ Export-ModuleMember -Function Set-AzKeyVaultSecretFromFile
 
 function Set-AzKeyVaultSecretAsBase64FromFile {
     param (
```

Comment:
Most of the time without space, so maybe change these in the file also without space? `param(`?

Reply:
Good one, fixed in [`901d049` (#424)](https://github.com/arcus-azure/arcus.scripting/pull/424/commits/901d0490716209e4a3a8a143f1037673457ca6bd)